From: Robert Luberda <robert@debian.org>
Date: Mon, 7 Mar 2011 20:20:10 +0100
Subject: 0019 Section of english(5)

Fix section of the english(5) man page. Patch taken from OpenSUSE's
ispell-3.3.02-languages.patch
---
 languages/english/english.5X |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/languages/english/english.5X b/languages/english/english.5X
index aa08fad..28c1e2b 100644
--- a/languages/english/english.5X
+++ b/languages/english/english.5X
@@ -56,7 +56,7 @@
 .\" Get rid of all old RCS log lines in preparation for the 3.1 release.
 .\"
 .\"
-.TH ENGLISH 4
+.TH ENGLISH !!MAN45SECT!!
 .SH NAME
 english \- flag format for English \fIispell\fP dictionaries
 .SH DESCRIPTION
@@ -66,7 +66,7 @@ supports 3 prefix and 14 suffix flags.
 For a detailed description of how
 .I ispell
 handles flags and capitalization, see
-.IR ispell (4).
+.IR ispell (!!MAN45SECT!!).
 This manual page only describes flags usable in dictionaries built
 using the
 .I english.aff
@@ -85,7 +85,7 @@ produced by placing the "D" flag on "we".
 .PP
 In the following list, an asterisk indicates that a flag participates in
 cross-product formation (see
-.IR ispell (4)).
+.IR ispell (!!MAN45SECT!!)).
 .PP
 The meaning of the prefix flags is as follows:
 .IP "*A"
@@ -248,7 +248,7 @@ if # .ne. s, x, z, h, or y, or (# = y and @ = a, e, i, o, or u)
 \&...@# --> ...@#s  as in bat --> bats
 or convey --> conveys
 .in -5m
-.I*P "P"
+.IP "P"
 .in +5m
 .ti -5m
 if @ .ne. a, e, i, o, or u,
-- 
